sway-lsp/src/capabilities/document_symbol.rs

```diff
@@ -21,30 +32,113 @@ pub fn to_symbol_information(tokens: &[Token], url: Url) -> Vec<SymbolInformatio
 
 #[allow(warnings)]
 // TODO: the "deprecated: None" field is deprecated according to this library
-fn create_symbol_info(token: &Token, url: Url) -> SymbolInformation {
+fn create_symbol_info(ident: &Ident, token: &TokenType, url: Url) -> SymbolInformation {
```

Comment:
```suggestion
fn symbol_info(ident: &Ident, token: &TokenType, url: Url) -> SymbolInformation {
```

Reply:
Nice good call, i've gone ahead and applied this change and the subsequent suggestions of removing the get_ and create_ prefexies.